quick-start/src/main/ui/app/components/flows-new/edit-flow/ui/step.component.ts

```diff
@@ -8,10 +9,27 @@ import { Component, Input, Output, EventEmitter, ViewChild } from '@angular/core
 })
 export class StepComponent {
   @Input() step: any;
-  constructor(
+  @Input() databases: any;
```

Comment:
What is "databases" refers to?

Reply:
databases is for the dropdown of databases the user can select from. ex staging, final, and jobs